community.html

```diff
@@ -4,7 +4,72 @@
 description: Meshery is community-built and welcomes collaboration!
 ---
 
-{% include partials/connect.html %} 
-
-
-<!-- <div style="position: relative; padding-bottom: calc(66.66666666666666% + 40px); height: 0;"><iframe src="https://demo.arcade.software/hgizzHHMzBDWwC7O9Fod?embed" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div> -->
\ No newline at end of file
+<section>
+  
+  <div class="container flex">
+    <div class="image flex">
+       <img style="max-width:100%; box-shadow: 0 5px 10px 1px rgba(0, 211, 169, 0.2);border: px black; border-radius: 10px;"
+          src="/assets/images/logos/Calendar Light Theme.svg"
+          data-logo-for-dark="/assets/images/logos/Calendar Dark Theme.svg"
+          data-logo-for-light="/assets/images/logos/Calendar Light Theme.svg"
+          alt="Calendar Image"
+          id="logo-dark-light"
+        />
+       
+      
+    </div>
+    <div class="text " style="width: auto; ">
+        <h1 style="font-family: 'Qanelas Soft', sans-serif; font-size: 3vmax; margin-bottom: 20px; margin-top: 20px;">Meshery Community <br>Meetings and Events</h1>
+        <p style="text-align: justify; margin-bottom: 20px; font-weight: 100; font-size: 14px;" >
+          Join Meshery’s meeting from anywhere in 
+        the world! We you missed on any of our events we 
+        have it recorded on our Youtube channel. You can also 
+        subscribe to our community calendar to stay up-to-date 
+        on all meetings and events.
+        </p>
+        
+        <div class="logoss" style="display: flex;align-items: center;justify-content: left;">

```

Comment:
```suggestion
        <div class="logos" style="display: flex;align-items: center;justify-content: left;">

```
- Consider renaming the classes, please. 
- Also, look for the globally available elements